lark/parsers/lalr_parser.py

```diff
@@ -11,11 +11,15 @@ class Parser:
     def __init__(self, parser_conf):
         assert all(r.options is None or r.options.priority is None
                    for r in parser_conf.rules), "LALR doesn't yet support prioritization"
-        self.analysis = analysis = LALR_Analyzer(parser_conf)
```

Comment:
this was holding back tons of objects used during the analysis that where no longer needed